bin/validate.py

```diff
@@ -787,7 +801,7 @@ def fix_pro_isoforms_in_gaf(gaf_file_to_fix: str, gpi_file: str, ontology_graph,
                                         "id": gpi_entry.get('id')}
 
     gafparser = GafParser(config=assocparser.AssocParserConfig(ontology=ontology_graph))
-    gafwriter = GafWriter(file=open(output_file_path, "w"), source="test", version=gafparser.version)
+    gafwriter = GafWriter(file=open(output_file_path, "w"), version="2.2")
```

Comment:
@dustine32 @kltm—this is the biggest change in this PR: Instead of taking the src-input GAF version to populate the output GAF, write out version 2.2.  

The rest of the PR is helpful too - e.g. creating temporary files instead of over-writing files, some test fixes of headers (e.g. consistently use GOC instead of a mix of "GOC" and "GO Central"), but this line is what fixed the chickens.